package.json

```diff
@@ -1,23 +1,17 @@
 {
   "name": "@ensdomains/dnsregistrar",
   "main": "dist/dnsregistrar.js",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "requires": true,
   "lockfileVersion": 1,
   "prettier": {
     "singleQuote": true
   },
   "dependencies": {
-    "@ensdomains/buffer": "0.0.6",
-    "@ensdomains/dnsprovejs": "^0.1.8",
-    "@ensdomains/dnssec-oracle": "^0.2.0",
+    "@ensdomains/buffer": "^0.0.6",
+    "@ensdomains/dnssec-oracle": "^0.4.0",
     "@ensdomains/ens": "^0.5.0",
-    "@ensdomains/root": "^0.1.0",
-    "@ensdomains/test-utils": "^1.3.0",
-    "dns-packet": "^5.0.4",
-    "eth-ens-namehash": "^2.0.8",
-    "js-sha3": "^0.5.7",
-    "web3": "1.0.0-beta.34"
```

Comment:
web3js is in use at https://github.com/ensdomains/dnsregistrar/blob/c3fde649767e435b248c0c9921ce0e4f8870cc4b/lib/dnsregistrar.js#L3 . Probably you can get rid of the `dnsregistrar.js` file all together

Reply:
Please update README to remove reference to `DNSRegistrarJs`